modules/vaos/app/services/vaos/jwt.rb

```diff
@@ -18,18 +18,33 @@ def payload
         sub: @user.icn,
         idType: 'ICN',
         iss: 'gov.va.vaos',
-        firstName: @user.first_name,
-        lastName: @user.last_name,
+        firstName: @user.mvi&.profile&.given_names&.first, # from MVI not SAML assertion
```

Comment:
Could this be a method named for the comment? e.g. `mvi_first_name` (same for last name)